velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -193,11 +192,11 @@ WindowFuzzer::generateSortingKeysAndOrders(
     std::vector<TypePtr>& types) {
   auto keys = generateSortingKeys(prefix, names, types);
   std::vector<SortingKeyAndOrder> results;
-  // TODO: allow randomly generating orders.
   for (auto i = 0; i < keys.size(); ++i) {
-    std::string order = "asc";
-    std::string nullsOrder = "nulls last";
-    results.push_back(SortingKeyAndOrder(keys[i], order, nullsOrder));
+    auto asc = vectorFuzzer_.coinToss(0.5);
+    auto nullsFirst = vectorFuzzer_.coinToss(0.5);
+    results.emplace_back(
+        SortingKeyAndOrder(keys[i], core::SortOrder(asc, nullsFirst)));
```

Comment:
nit: No need to use SortingKeyAndOrder constructor when using emplace_back().

Reply:
Fixed, thanks.